ui/src/app.jsx

```diff
@@ -48,14 +48,17 @@ export default React.createClass({
     '/virtual_school': 'virtualSchool',
     
     '/challenge/:id': 'challenge',
-    '/message_popup': 'messagePopup',
-    '/message_popup/exploration': 'messagePopupExploration',
-    '/message_popup/evaluations/:id': 'messagePopupEvaluation',
-    '/message_popup/scoring': 'messagePopupScoring',
-    '/message_popup/author/questions' : 'messagePopupAuthorQuestions',
-    '/message_popup/author/questions/new' : 'messagePopupAuthorQuestionsNew',
-    '/message_popup/author/questions/:id' : 'messagePopupAuthorQuestionsEdit',
-    '/message_popup/audio_flow_prototype': 'messagePopupAudioFlowPrototype',
+
+    '/message_popup/*': 'messagePopupRedirect',
+    '/teachermoments': 'messagePopup',
```

Comment:
I'm not sure this matters (pending on deploying this to a new domain), but I kept this `teachermoments` to match what the domain name will be.  I'm assuming that when we deploy this as a standalone experience we'll remove these prefixes in the routes altogether.